src/Tests/Unit/Pipeline/Bank/VisaCal/Login/VisaCalLoginPreAction.test.ts

```
@@ -0,0 +1,98 @@
+
+/** Mock only waitUntilIframeFound — spread actual module for other exports. */
+const ACTUAL = await import('../../../../../../Common/ElementsInteractions.js');
+jest.unstable_mockModule('../../../../../../Common/ElementsInteractions.js', () => ({
+  ...ACTUAL,
+  /**
+   * Call the predicate with MOCK_FRAME then return it.
+   * @param _page - Unused page.
+   * @param predicate - Frame predicate to exercise.
+   * @returns Resolved mock frame.
+   */
+  waitUntilIframeFound: (_page: Page, predicate: (_f: Frame) => boolean): Promise<Frame> => {
```

ESLint

[error] '_f' is defined but never used.

(no-unused-vars)